rotkehlchen/tests/api/test_balances.py

```diff
@@ -911,3 +912,32 @@ def test_balances_behaviour_with_manual_current_prices(rotkehlchen_api_server, e
         rdn_result = result['assets']['eip155:1/erc20:0x255Aa6DF07540Cb5d3d297f0D0D4D84cb52bc8e6']
         assert rdn_result['amount'] == '5'
         assert rdn_result['usd_value'] == '150.0'
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x7e574e063903b1D6DFf54A9C8B1260e6E068d35e']])
+def test_blockchain_balances_refresh(rotkehlchen_api_server: 'APIServer', ethereum_accounts):
+    """Checks that blockchain balances are refreshed properly when the endpoint is called"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+
+    account_balance = {ethereum_accounts[0]: BalanceSheet(
```

Comment:
What is this test doing? You are both mocking and VCRing. Don't. Do one or the other. Otherwise this is going to be impossible to maintain in the future.

Just use VCR and a real balance example where this problem you found is happening. 

**Make sure the test fails** without your fix.